CONTRIBUTING.md

```diff
@@ -33,7 +33,7 @@ The goal of the community is to develop a cloud native edge computing platform b
 # Getting started
 
 - Fork the repository on GitHub
-- Read the [setup](https://github.com/kubeedge/kubeedge/blob/master/docs/setup/setup.md) for build instructions.
+- Read the [setup](./docs/setup/index.md) for build instructions.
```

Comment:
It is not related to PR title and would be fixed in #1848, so remove this change please.